data-protocols/ids/ids-api-multipart-endpoint-v1/src/main/java/org/eclipse/dataspaceconnector/ids/api/multipart/IdsMultipartApiServiceExtension.java

```diff
@@ -147,6 +148,7 @@ private void registerControllers(ServiceExtensionContext serviceExtensionContext
         objectMapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
         objectMapper.configure(DeserializationFeature.ACCEPT_SINGLE_VALUE_AS_ARRAY, true);
         objectMapper.registerSubtypes(IdsConstraintImpl.class);
+        objectMapper.registerModule(new XmlGregorianCalendarModule());
```

Comment:
I see this `objectMapper` definition nearly equal to the one in `IdsMultipartDispatcherServiceExtension` and in `AbstractMultipartDispatcherIntegrationTest`. Could be extracted in an `ObjectMapperFactory` in the `ids-core` module.

Reply:
Good point, I'll add an `ObjectMapperFactory`, will make the extensions a little cleaner.

Reply:
Good point, I'll add an `ObjectMapperFactory`, will make the extensions a little cleaner.